.github/workflows/lint.yml

```diff
@@ -0,0 +1,34 @@
+name: Linting
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Npm install and build
+      # A "full" install is executed, since `npm ci` does not always exit
+      # with an error status code if the lock file is inaccurate.
+      #
+      # See: https://github.com/WordPress/gutenberg/issues/16157
+      run: |
+        npm install
+        npm run test:create-block
```

Comment:
Do you plan to remove: https://github.com/WordPress/gutenberg/blob/595989a23b937931f69f494bf78eb9a91228b1a3/.github/workflows/create-block.yml?

Reply:
this was unintended  copy/paste :P

Reply:
Aside: one thing I discovered with the workflow for create block is that using exclude patterns is a bit problematic. If you have a PR where you change files that trigger action and then push a commit with excluded files then the check gets removed from PR. So, in theory, you could push broken code, then push unrelated excluded changes and it won't complain anymore 😅